src/FixedDataTable.js

```diff
@@ -751,8 +764,15 @@ var FixedDataTable = createReactClass({
     /*number|string*/ columnKey,
     /*object*/ event
   ) {
+
+    var nowMs = Date.now();
+    if (this.state.lastColumnResizeStart && nowMs - this.state.lastColumnResizeStart < DOUBLE_CLICK_TIMEOUT) {
+      (this.props.onColumnResizeDoubleClick || emptyFunction)(columnKey);
+    }
```

Comment:
Theres no reason to re-invent the wheel here. You should have a separate callback passed that uses the native onDoubleClick event handler